permissions/plugin-depgraph-view.yml

```diff
@@ -4,4 +4,7 @@ github: "jenkinsci/depgraph-view-plugin"
 paths:
 - "org/jenkins-ci/plugins/depgraph-view"
 developers:
-- "wolfs"
```

Comment:
I don't think that we typically remove the old contributors...

Reply:
Yeah, please leave me in if that doesn't bother you.

Reply:
Of course not, sorry for my misunderstanding